recipes/minvar/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: minvar 
+  version: "1.2a3"
+
+build:
+  number: 0
+  skip: True # [not py35]
```

Comment:
Sure. At least at the beginning, though, I will still skip py27. I haven't written minvar with support for it. What is your policy for tools that depend on something that is not (yet) on bioconda?